src/ape/managers/project/manager.py

```diff
@@ -447,16 +447,18 @@ def _get_contract(self, name: str) -> Optional[ContractContainer]:
 
         return None
 
-    # @property
-    # def meta(self) -> PackageMeta:
-    #     return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
+    @property
+    def meta(self) -> PackageMeta:
+        return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
+
+    def publish_manifest(self):
+        manifest = self.manifest.dict()
+        if not manifest["name"]:
+            raise ProjectError("Need name to release manifest")
+        if not manifest["version"]:
+            raise ProjectError("Need version to release manifest")
+        breakpoint()
 
-    # def publish_manifest(self):
-    #     manifest = self.manifest.dict()
-    #     if not manifest["name"]:
-    #         raise ProjectError("Need name to release manifest")
-    #     if not manifest["version"]:
-    #         raise ProjectError("Need version to release manifest")
     #     TODO: Clean up manifest and minify it
```

Comment:
You can remove this `TODO`, `ethpm-types` already performs this cleanup internally when you call `manifest.dict()` (without any additional kwargs)